src/static/css/skip-nav.less

```diff
@@ -0,0 +1,24 @@
+#skip-nav {
+    padding: 10px;
```

Comment:
They already are following the property order: Box Model, Position, Alphabetical order. Is something off?